template/web/app/api/rpc/route.ts

```diff
@@ -0,0 +1,30 @@
+import { NextRequest, NextResponse } from 'next/server';
+
+const rpcUrl = process.env.NEXT_PRIVATE_RPC_URL;
+
+export async function POST(req: NextRequest) {
```

Comment:
@0xAlec @cpcramer can we add some docs that explain more what's going on here.

Remember, many junior will use BOAT without clear context of the internal. So it's important to be as verbose as possible.